src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -132,6 +174,19 @@
   }
 }
 
+.pf-c-context-selector__toggle-button-icon {
+  position: relative;
+}
+
+.pf-c-context-selector__toggle-icon {
+  margin-right: var(--pf-c-context-selector__toggle-icon--MarginRight);
+  margin-left: var(--pf-c-context-selector__toggle-icon--MarginLeft);
```

Comment:
It looks like these margin styles are already included here - https://github.com/patternfly/patternfly/pull/4095/files#diff-f6fa17cc8d90aa2f9022c472197706093dbe9f9330d00d4ff2d2dd5892e0e450R163-R166

However, I think moving them out of the toggle class selector is a great idea! There is no need for those styles to be nested the way they are. Can you remove those lines in favor of this block you added?

Reply:
Looks like the other block for `.pf-c-context-selector__toggle-icon` is still in the CSS and doesn't need to be since you've duplicated it here.